vmm/src/vm.rs

```diff
@@ -1078,7 +1099,7 @@ pub fn test_vm() {
             VcpuExit::Epr => {}
             VcpuExit::SystemEvent => {}
             VcpuExit::S390Stsi => {}
-            VcpuExit::IoapicEoi => {}
+            VcpuExit::IoapicEoi(_vector) => {}
```

Comment:
Is this a rebase error?

Reply:
Now, it is rebased. Should compile fine.

Reply:
Rease is done now.